### PR TITLE
Adapting Awesome Books project to work with modules

### PR DESCRIPTION
**Pull request by Svartstein**
Refractor of the Awesome Books Proyect to use JS modules.

### CODE REVIEW

File: modules/books.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      for (let i = 0; i < books.length; i += 1) {
+        if (item.id === this.id) {
+          // console.log(`book with id = ${this.id} identified`);
+          return false;
+        }
+        return true;
+      }
+    });
+    localStorage.setItem('BOOkS', JSON.stringify(books));
+    // console.log('this is the books array:\n' + localStorage.getItem('BOOkS'));
```
Comment (julie-ify) [inline modules/books.js]: - [ ] Kindly remove these commented console logs as they look untidy and can make your code had to read.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  // console.log("this book's ID will be "+n)
+  const newBook = new Book(n, inpTitle.value, inpAuthor.value);
+  // console.log('new book created:\n'+JSON.stringify(newBook));
+  newBook.Add();
+  // console.log('books added to local storege');
+  checkStorage();
+  printBooks();
+  // console.log('books printed');
+  // console.log(storage);
+});
```
Comment (julie-ify) [inline index.js]: - [ ] Remove these console logs

File: modules/printer.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      <p>${bk.title} by ${bk.author}</p>
+      <button class="erase-book">Erase</button>
+    </div>
+    `;
+  }
+  listSection.innerHTML = printed;
+  document.querySelectorAll('.erase-book').forEach((element) => {
+    element.addEventListener('click', (e) => {
+      const bookId = parseInt(e.target.parentNode.id, 10);
+      // console.log(`the book with id = ${bookId} will be deleted`);
```
Comment (julie-ify) [inline modules/printer.js]: - [ ] Remove the console log

File: index.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
+// eslint-disable-next-line
+import Book from './modules/books.js';// eslint-disable-next-line
```
Comment (julie-ify) [inline index.js]: - [ ] Kindly enable linters here and fix the resulting errors. Linters help us write standardized code and meet recommended best practices

File: modules/books.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  Add = () => {
+    books.push({ id: this.id, title: this.title, author: this.author });
+    // console.log('this is the books array:\n'+JSON.stringify(books));
+    localStorage.setItem('BOOkS', JSON.stringify(books));
+    // console.log('saved in local storage:');
+    checkStorage();
+  };
+
+  Delete = () => {
+    books = JSON.parse(localStorage.getItem('BOOkS'));// eslint-disable-next-line
```
Comment (julie-ify) [inline modules/books.js:25]: - [ ] Kindly enable linter

File: modules/printer.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line
```
Comment (julie-ify) [inline modules/printer.js]: - [ ] Kindly enable linter here

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Swarzstein,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
- [ ] You are required to display date and time using the luxon package under the right side of the menu bar. Kindly do that

  gkg

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — COMMENTED** [review]
Hi @Swarzstein 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that whose linters fail. Currently, the Elslint that is supposed to standardize your JS code is failing. Kindly fix this before requesting for another review as it is a core project requirement.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by omarsalem7 — APPROVED** [review]
Hi @Swarzstein  😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**